README.md

```diff
@@ -34,6 +34,17 @@ DATABASE_URL=postgres://jrob@localhost:5432/floworky
 * Assertion Library: [Chai](http://chaijs.com/)
 * Test Spies, Stubs and Mocks: [Sinon](http://sinonjs.org/)
 * Test Coverage: [Istanbul](https://github.com/gotwarlost/istanbul)
+* Static Type Checking: [Flow](https://flowtype.org)
+
+### Flow
+Use the flow pragma ```//@flow``` on a given file to check the file for type errors. After selecting the files, either use the cli script ```npm run flow``` on the terminal or use the Nuclide IDE to use flow for type checking.
+For more complete documentation about flow and annotations please checkout the [flow website](https://flowtype.org/docs/quick-reference.html)
+
+### Configure Atom w/ Nuclide for Flow
```

Comment:
Let's move this section under `Setting up Development Environment`:

```md
#### (Optional) Configure Atom with Nuclide for Flow
Floworky uses flow for static type checking, which can be manually run using `npm run flow`.  If you are using Atom to contribute, you can [use Nuclide](https://nuclide.io/docs/editor/setup/#quick-install) for type annotation information within your editor.
```

This text provides enough information for would-be developers to set up Atom, and refers to appropriate documentation for more information.